server/routes/appointments/feedback/actionPlanSessions/checkYourAnswers/actionPlanPostSessionFeedbackCheckAnswersPresenter.ts

```diff
@@ -23,13 +23,30 @@ export default class ActionPlanPostSessionFeedbackCheckAnswersPresenter extends
     : `/service-provider/action-plan/${this.actionPlanId}/appointment/${this.actionPlanAppointment.sessionNumber}/post-session-feedback/submit`
 
   get backLinkHref(): string {
-    if (this.draftId) {
-      return this.actionPlanAppointment.appointmentFeedback.attendanceFeedback.attended === 'no'
-        ? `/service-provider/action-plan/${this.actionPlanId}/appointment/${this.actionPlanAppointment.sessionNumber}/post-session-feedback/edit/${this.draftId}/attendance`
-        : `/service-provider/action-plan/${this.actionPlanId}/appointment/${this.actionPlanAppointment.sessionNumber}/post-session-feedback/edit/${this.draftId}/behaviour`
+    if (this.actionPlanAppointment.appointmentFeedback.attendanceFeedback.didSessionHappen === true) {
```

Comment:
can this be `if (this.actionPlanAppointment.appointmentFeedback.attendanceFeedback.didSessionHappen) {`